core-backend/wasmi/src/funcs/mod.rs

```diff
@@ -51,14 +51,21 @@ use wasmi::{
     AsContextMut, Caller, Func, Memory as WasmiMemory, Store,
 };
 
-#[derive(Debug, derive_more::Display, Encode, Decode)]
+// TODO: change it to u32::MAX (issue #2027)
+const PTR_SPECIAL: u32 = i32::MAX as u32;
```

Comment:
Is it possible to move the constant into common module?

Reply:
Yes possible)) I think it's better to make it in future refactoring